docs/manuals/azure-setup-manual.md

```diff
@@ -0,0 +1,24 @@
+# Azure Registration Manual ZGW Office Add-In
+
+## Inleiding
+This user guide is intended for the administrators of the Azure environment of the respective municipality.
+This guide will describe how the add-in can be registered within the Azure environment.
+
+## The Manual
+To register a add-in within your azure environment you can follow the official manual of microsoft [here](https://learn.microsoft.com/en-us/office/dev/add-ins/develop/register-sso-add-in-aad-v2). Using this manual there are a few inputs that which can be filled in like the following
+
+# Register the add-in with Microsoft identity platform
+The name of the add-in is "zgw-office-add-in".
+
+The redirect URI can be the following.
+![Redirect-URI](./images/azure-registratie/redirect_uri.png)
+
+# Expose a web API
+The application ID URI is "api//zgw-office-addin-dev-frontend.dimpact.lifely.nl/proxy/client_id".
+Replace client_id with your own generated client_id.
```

Comment:
Move this one above the application ID URI line, this is also above that in the xml file.
And I would possibly add a part of the manifest xml to show the entries.